File: despatch.WEB/app/src/swagger/models/user-rooms-request.ts
/* tslint:disable */
import { BaseEntityRequest } from './base-entity-request';
import { RoomRequest } from './room-request';
import { UserRequest } from './user-request';
export interface UserRoomsRequest extends BaseEntityRequest{
  missedMessagesCount?: string;
  room?: RoomRequest;
  roomId?: string;
  user?: UserRequest;
  userId?: string;
}

File: despatch.Domain/Models/Default/Request/UserRoomsRequest.cs
namespace despatch.Domain.Models.Default.Request
{
    public class UserRoomsRequest : BaseEntityRequest
    {
        public string MissedMessagesCount { get; set; }

        public string UserId { get; set; }
        public string RoomId { get; set; }

        public virtual UserRequest User { get; set; }
        public virtual RoomRequest Room { get; set; }

    }
}
